src/settings.cpp

```diff
@@ -48,6 +48,7 @@ void SettingsManager::ReadSettings() {
     SMBOOL("gui", "save_state", SaveState);
     SMBOOL("gui", "stock_emojis", ShowStockEmojis);
     SMBOOL("gui", "unreads", Unreads);
+    SMBOOL("gui", "HideToTray", HideToTray);
```

Comment:
```suggestion
    SMBOOL("gui", "hide_to_tray", HideToTray);
```
for consistency with the other settings names